rhconsulting_widgets.rake

```diff
@@ -20,7 +20,7 @@ class MiqWidgetsImportExport
 private
 
   def export_widgets(export_dir)
-    custom_widgets = MiqWidget.find_all_by_read_only("false")
+    custom_widgets = MiqWidget.where(read_only => "false")
```

Comment:
I think a semicolon is missing here as well:  It should be MiqWidget.where(**:read_only** => "false")